### PR TITLE
Features/action file transfer2 linux transformation

### PR DESCRIPTION
**Pull request by uzanh**

### CODE REVIEW

File: Ginger/Ginger/Actions/ActionEditPages/ActFileTransferEditPage.xaml.cs [hunk 1]
```diff
@@ -71,15 +71,15 @@ private void BrowsePCPathButton_Click(object sender, RoutedEventArgs e)
...
             if (dlg.ShowDialog() == System.Windows.Forms.DialogResult.OK)
             {
                 // replace Absolute file name with relative to solution
-                string FileName = dlg.FileName.ToUpper();
-                if (FileName.Contains(SolutionFolder))
+                string FileName = dlg.FileName;
+                if (FileName.ToUpper().Contains(SolutionFolder.ToUpper()))
                 {
-                    FileName = FileName.Replace(SolutionFolder, @"~\");
+                    FileName = @"~\" + FileName.Remove(0, SolutionFolder.Length);
```
Comment (MeniKadosh1) [inline Ginger/Ginger/Actions/ActionEditPages/ActFileTransferEditPage.xaml.cs:82]: why you preferred Remove than replace?

Let's use centalized function (need to check if exist and if not so to add one) which deals with relative path handling instead doing it inside the Action code

File: Ginger/GingerCore/GingerCore.csproj [hunk 2]
```diff
@@ -885,7 +884,7 @@
       <Version>1.8019.4006.503</Version>
     </PackageReference>
     <PackageReference Include="SSH.NET">
-      <Version>2016.1.0</Version>
+      <Version>2020.0.1</Version>
```
Comment (MeniKadosh1) [inline Ginger/GingerCore/GingerCore.csproj:887]: did you tested all possible impact due to upgrade also on Unix driver and other depended code?

File: Ginger/Ginger/Actions/ActionEditPages/ActFileTransferEditPage.xaml.cs (hunk 1 above)
Comment (uzanh) [reply Ginger/Ginger/Actions/ActionEditPages/ActFileTransferEditPage.xaml.cs:82]: 1. not to be depend on case sensitive
2. ok, we have one

File: Ginger/GingerCore/GingerCore.csproj (hunk 2 above)
Comment (uzanh) [reply Ginger/GingerCore/GingerCore.csproj:887]: check agent , what we have more?

**Review by MeniKadosh1 — APPROVED** [review]

**Pull request merged**